components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -465,6 +472,25 @@
   }
 
   //#endregion consecutive events
+
+  //#region expand/collapse
+
+  let sections: Partial<Sections> = {};
+
+  function establishSections() {
+    Object.values(SectionNames).forEach((name, iter) => {
+      sections[name] = {
+        expanded: iter === 0 ? true : false,
```

Comment:
nit 
```suggestion
        expanded: iter === 0,
```

Reply:
I am usually pro-terse but this is a spot where I think the long-form ternary helps me understand what's going on at a glance faster.